server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HGET.java

```diff
@@ -0,0 +1,41 @@
+package org.infinispan.server.resp.commands.hash;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.multimap.impl.EmbeddedMultimapPairCache;
+import org.infinispan.server.resp.Consumers;
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespErrorUtil;
+import org.infinispan.server.resp.RespRequestHandler;
+import org.infinispan.server.resp.commands.Resp3Command;
+
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * `<code>HGET key field</code>` command.
+ * </p>
+ * Find the value associated with field in the hash stored at key.
+ *
+ * @since 15.0
+ * @see <a href="https://redis.io/commands/hget/">Redis Documentation</a>
```

Comment:
I think we should be consistent in thejava doc. The @see seems better to me. I will refactor other commands in other PR